docs/data/material/components/links/links.md

```diff
@@ -51,5 +51,16 @@ Here is a [more detailed guide](/material-ui/guides/routing/#link).
   Instead, use [specific descriptions](https://developers.google.com/web/tools/lighthouse/audits/descriptive-link-text).
 - For the best user experience, links should stand out from the text on the page. For instance, you can keep the default `underline="always"` behavior.
 - If a link doesn't have a meaningful href, [it should be rendered using a `<button>` element](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/HEAD/docs/rules/anchor-is-valid.md).
+  The demo below illustrates how to properly link with a `<button>`:
 
 {{"demo": "ButtonLink.js"}}
+
+### Keyboard accessibility
+
+- Interactive elements should receive focus in a coherent order when the user presses the <kbd class="key">Tab</kbd> key.
+- Users should be able to open a link by pressing <kbd class="key">Enter</kbd>.
+
+### Screen reader accessibility
+
+- When a link receives focus, screen readers should announce a descriptive link name.
+  If the link opens in a new window or browser tab, add help text in the code to inform screen reader users—for example, _"To learn more, visit the **About** page which opens in a new window."_
```

Comment:
Does anyone have a citation for this? @mnajdova @michaldudak  I tried searching the W3C APG but couldn't find anything to explain exactly _how_ you would "add help text in the code" in this instance. I would prefer to be more specific here.

Reply:
Using `aria-label`. See https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA8#:~:text=The%20aria%2Dlabel%20attribute%20provides,used%20instead%20of%20aria%2Dlabel%20. I agree, maybe we should replace "add help text" with "add aria-label".

Reply:
Thanks! I figured this would be an aria-label but I wasn't sure.